documentation/FlowManager.md

```diff
@@ -109,6 +109,20 @@ payload structure to start a matchMaking process.
 * **return:** Return http status code 200 when Cloud Based Flow Manager itself is running. Otherwise, return http status
  code 500.
 
+### Retrieve the revision of the Cloud Based Flow Manager (GET /revision)
+
+* **description**: Serve the full `SHA256` of the revision of the source code repository used by this deployment of the
```

Comment:
It would be helpful to describe how the retrieved revision number will be used by the untrusted local flow manager.

Reply:
It's a general query. 

Yes, the original motivation was for fetching the latest solutions registry from github.  However, the other day, @javihernandez [asked](https://raisingthefloor.slack.com/archives/C9PCPPUF8/p1579809274004900) in the `ops` channel in slack if there was a quick way find out what version of universal was currently deployed.  Currently, there is a rake task that can be run, or you look up the logs from `ci-service` when it created docker image.

That said, if you really think it's important to provide the original rationale, I can do that.

Reply:
Good point, @klown, it probably makes more sense to document in the Solution Registry documentation to describe how it gets fetched and populated in your pull request for a later stage of this work.

Reply:
Sure, but I prefer to add that as part of #840, which does the actual fetching/populating.